src/esp/gfx/LightSetup.cpp

```diff
@@ -76,12 +76,34 @@ LightSetup getLightsAtBoxCorners(const Magnum::Range3D& box,
                     {{box.backBottomLeft(), w}, lightColor},
                     {{box.backBottomRight(), w}, lightColor}};
 }
+
 LightSetup getDefaultLights() {
-  return LightSetup{{{0.0, -0.5, -1.0, 0.0}, {2.4, 2.4, 2.4}},    // forward
-                    {{0.0, -0.5, 1.0, 0.0}, {2.4, 2.4, 2.4}},     // backward
-                    {{1.0, -1.0, 0.0, 0.0}, {2.0, 2.0, 2.0}},     // top left
-                    {{-1.0, -1.0, -1.0, 0.0}, {2.0, 2.0, 2.0}},   // top right
-                    {{0.0, 1.0, 0.0, 0.0}, {0.64, 0.64, 0.64}}};  // bottom
+  return LightSetup{
+      {{0.0, -0.1, -1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // -z
+      {{0.0, -0.1, 1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // +z
+      {{-1.0, -0.1, 0.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // -x
+      {{+1.0, -0.1, -1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Global},  // +x
+      {{0.0, -1.0, 0.0, 0.0},
+       {0.6, 0.6, 0.6},
+       LightPositionModel::Global},  // downward
+
+      // camera-relative
+      {{0.0, -0.1, -1.0, 0.0},
+       {0.3, 0.3, 0.3},
+       LightPositionModel::Camera},  // forward
+      {{0.0, -0.3, 1.0, 0.0},
+       {0.5, 0.5, 0.5},
+       LightPositionModel::Camera},  // backward (for glancing speculars)
```

Comment:
I would avoid emitting light from the camera by default here.

Reply:
This is done specifically so that surface details such as normal or clearcoat texture/specular can be readily seen in the new PBR shader regardless of camera orientation (i.e. demo screen captures).

Reply:
Like I said below, I can remove these in favor of a custom lighting config added to /data that folks could just load up to take screen shots and such.